src/components/JobActionsPopover.vue

```diff
@@ -89,17 +93,31 @@ export default defineComponent({
         await this.store.dispatch('user/updatePinnedJobs', { pinnedJobs: [...pinnedJobs] });
       }
       this.closePopover();
+    },
+    async runJobNow(job: any) {
+      if(job) {
+        const { Clipboard } = Plugins;
+        const jobDetails = `jobId: ${job.jobId}, jobName: ${this.getEnumName(job.systemJobEnumId)}, jobDescription: ${this.getEnumDescription(job.systemJobEnumId)}`;
+
+        await Clipboard.write({
```

Comment:
Why we have added logic to copy the job details on clicking run now?

Reply:
Sir it was mentioned in the ticket itself.
Please have a look - https://app.clickup.com/t/364ttgf

Reply:
Ok, it means that we need to `copy the job`(use the same job for run now) and just update the recurrence count, we don't need to copy job details.